app/views/reference/comments.njk

```diff
@@ -18,6 +18,7 @@
 {% endblock %}
 
 {% block primary %}
+  <p class="govuk-body"> You have to complete the reference in one sitting. You cannot save it and come back to it later.</p>
```

Comment:
Is saying ‘you have’ a bit too direct, perhaps going again the softer tone we are aiming for? Especially as it’s then followed by ‘cannot’.

Probably want to stay away from the warning component; that tends to be used in very rare circumstances. If this continues to prove a problem, maybe we can look at other approaches? 

```suggestion
  <p class="govuk-body">Complete your reference in one sitting. You cannot save it and come back to it later.</p>
```